### DISCUSSION TITLE
api 리턴 메세지 정적인 방향으로 수정 제안

### DISCUSSION THREAD

**Discussion by jis-kim** [GhostPangPang/GhostPong#53] [Polls]
현재 친구 신청 api 에서 성공할 경우 
```js
{
	"message": "<nickname> 에게 친구신청이 완료되었습니다."
}
```
인데 id 로 친구 신청할 경우 닉네임을 구하기 위해 DB 를 또 한 번 갔다와야 합니다.. (id 로 닉네임 구하기)

```js
{
	"message": "친구신청이 완료되었습니다."
}
```

로 변경 제안합니다!
비슷하게 동적인 메세지는 다 바뀌었으면 좋겠습니다. 의견 있으면 댓글 부탁드려요!

**Comment by jis-kim** [comment]
@anso33 @Kimhan-nah @nyeoni @aLVINlEE9

**Reply by anso33** [reply]
와 이거 알림이 안와요 쩝....
-> 문제 해결.....ㅋㅎㅋㅎ

**Reply by Kimhan-nah** [reply]
Unwatch -> Custom -> Disccusion 등등 체크하면 알림 받기 성공!

**Comment by nyeoni** [comment]
전 좋습니당 ㅎㅎ >_<

**Reply by jis-kim** [reply]
다수의 동의로 resolve 하겠습니다! 없애는 방향으로 결정~
